### ISSUE TITLE
Universal Render Pipeline

### ISSUE THREAD

**Issue by gegagome** [unitycoder/UnityMobilePaint#51]
Any reason URP caused Unity Coder/Canvas Default (Shader) to show pink?

I upgraded the shaders and they still show pink.

Thanks

The only scene where that shader displays pink is 'scene_MobilePaint_NewUI_Default'

All other scenes can be used as expected.

**Comment by gegagome** [comment]
Yeah, I installed URP on a brand new mobile paint project and Unity Coder/Canvas Default (Shader) started showing pink.

On my project I removed URP from the Edit/Project Settings/Graphics and it worked as expected.

For now I won't have any of the crappy shader graph shader I was working on but I was wondering if you knew of a solution and thanks!

**Comment by gegagome** [comment]
Using 2020.1.8f1

**Comment by unitycoder** [comment]
URP not supported, so would have to manually upgrade those shaders (or try any other shader from URP)

since that shader doesnt seem have anything special/custom features:
https://github.com/unitycoder/UnityMobilePaint/blob/master/Assets/MobilePaint/Materials/Shaders/CanvasDefault.shader

**Comment by gegagome** [comment]
I have 0 shader experience. The funny thing is that the other shaders work on and this one doesn't.

**Comment by unitycoder** [comment]
i think could try any unlit material if it looks ok.

**Comment by gegagome** [comment]
Are you planning on updating to URP? I made the mistake on upgrading my project to URP and had to use a backup to restore things and other things disappeared as well.

Thanks for your help

**Comment by unitycoder** [comment]
not likely.. there's also probably many better plugins available for drawing into texture.
maybe best option is to try using AI to convert shaders.
